html/index.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <script
+      src="https://kit.fontawesome.com/e744810f7a.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="/css/styles.css" />
+    <title>CEOs International Community Home Page</title>
+  </head>
+  <body>
+    <section class="first-bar-section">
+      <div class="first-bar-container">
+        <ul class="first-bar">
+          <li>
+            <i class="fab fa-facebook-f"></i>
+          </li>
+          <li>
+            <i class="fab fa-twitter"></i>
+          </li>
+          <li>
+            <h4>English</h4>
+          </li>
+          <li>
+            <h4>My Page</h4>
+          </li>
+          <li>
+            <h4>Logout</h4>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <header>
+      <div class="nav-container">
+        <div class="desktop-logo">
+          <a href="index.html"
+            ><img src="/assets/cic_logoReal.png" alt="logo"
+          /></a>
+          <div class="hamburger nav-toggle" id="menu">
+            <i class="fas fa-bars fa-3x"></i>
+          </div>
+          <div class="close-menu" id="close">
+            <p>&times;</p>
+          </div>
+        </div>
+
+        <nav class="webpages-menu" id="mob-nav">
+          <ul class="menu-controls">
+            <li class="webpages">
+              <a href="index.html">Home</a>
+            </li>
+            <li class="webpages">
+              <a href="about.html">About</a>
+            </li>
+            <li class="webpages">
+              <a href="#">Program</a>
+            </li>
+            <li class="webpages">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="webpages">
+              <a href="#">News</a>
+            </li>
+          </ul>
+
+          <div class="company-name">
+            <a href="index.html">CIC Campaign</a>
+          </div>
```

Comment:
- [ ] The navbar items should be in a single line. The CIC campaign overlaps to the next line displacing news nav item ⏬ 
![ci](https://user-images.githubusercontent.com/33067256/129425717-a5b1f7f2-15ac-435e-bab2-9e3634387979.png)
Kindly work on this